### PR TITLE
Transitioned from requirements.txt to Pipenv

### PR DESCRIPTION
**Pull request by dolphingarlic**
https://codein.withgoogle.com/dashboard/task-instances/4742782569676800/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,3 +90,5 @@ of the repository to update all the encoded files.
 
 You can also do `sudo make install` to install to `/usr/local/bin/apertium-init`
 or e.g. `PREFIX=$HOME/local make install` to install to `$HOME/local/bin/apertium-init`.
+
+To install dev dependencies, run `pipenv install --dev`
```
Comment (sushain97) [inline README.md]:
```suggestion
Use `pipenv install --dev` to install the requirements required for development, e.g. linters.
```

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+certifi = "*"
```
Comment (sushain97) [inline Pipfile]: A ton of these are transitive dependencies. Please remove all of those that are. See apertium-streamparser and apertium-apy for guidance.

Comment (dolphingarlic) [reply Pipfile]: Done

Comment (dolphingarlic) [reply Pipfile]: Is this why the requirements.txt model is bad?

Comment (sushain97) [reply Pipfile]: yeah, among other things :)

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+[dev-packages]
+flake8 = "*"
+flake8-commas = "*"
+flake8-polyfill = "*"
+flake8-quotes = "*"
+pep8-naming = "*"
+coverage = "*"
+coveralls = "*"
+mypy = { version = "*", implementation_name = "!= 'pypy'" }
```
Comment (sushain97) [inline Pipfile]: We don't need this marker since this lib doesn't get tested on `pypy`.

**Pull request merged**